src/main/resources/org/jenkinsci/plugins/ewm/nodes/DiskNode/help-localRootPath.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The mounting point to the physical disk.
+</div>
```

Comment:
The user may not have a physical disk (it might be a symbolic link again). So perhaps the help should say "The mounting point from this node to the disk"